lib/auth_token.rb

```diff
@@ -0,0 +1,13 @@
+class AuthToken
+  def self.encode(payload)
+    payload[:exp] = 24.hours.from_now.to_i
+    JWT.encode payload, Rails.application.secrets.secret_key_base, "HS512"
+  end
+
+  def self.decode(token)
+    JWT.decode(token, Rails.application.secrets.secret_key_base,
+    true, { algorithm: "HS512" })[0]
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.